modules/tests/jvm/src/test/scala/lucuma/catalog/TargetImportSuite.scala

```diff
@@ -32,8 +34,38 @@ class TargetImportFileSuite extends CatsEffectSuite:
         .through(TargetImport.csv2targets)
         .compile
         .toList
+        // .flatTap(x => IO(pprint.pprintln(x)))
```

Comment:
Commented code

Reply:
It is intentional, I uncomment it sometimes for testing